<launch>
  <!--using 'find' command for portability-->
  <node pkg="camera_calibration" type="images_pub" name="images_pub" output="screen">
    <param name="images_path" type="string" value="$(find camera_calibration)/images/" />
  </node>

  <!--respawn node in case of crash-->
  <node pkg="camera_calibration" type="camera_calibration_node" name="calibration" output="screen" respawn="true">
    <param name="params_path" type="string" value="$(find camera_calibration)/params/camera_params.yaml" />
  </node>
</launch>
